cob_bringup/robots/cob4-10.xml

```diff
@@ -35,15 +35,15 @@
 				<arg name="robot" value="$(arg robot)"/>
 				<arg name="pc" value="$(arg cob4-10-b1)"/>
 			</include>
-
-			<include file="$(find cob_bringup)/drivers/sick_flexisoft.launch">
-				<arg name="robot" value="$(arg robot)"/>
-			</include>
-			<include file="$(find cob_bringup)/drivers/bms.launch">
-				<arg name="robot" value="$(arg robot)"/>
-			</include>
 		</group>
 
+		<include file="$(find cob_bringup)/drivers/sick_flexisoft.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="sim" value="$(arg sim)"/>
+		</include>
+		<include file="$(find cob_bringup)/drivers/bms.launch">
+			<arg name="robot" value="$(arg robot)"/>
+		</include>
```

Comment:
at the moment there is no such distinction between sim/hw in bms.launch
@souravran do you already have such a fake_bms script?